client/my-sites/domains/domain-management/settings/cards/domain-redirect-card.tsx

```diff
@@ -119,32 +142,27 @@ class DomainRedirectCard extends Component {
 	}
 }
 
+const withRedirectAsKey = createHigherOrderComponent(
+	( Wrapped ) => ( props ) => {
+		const selectedSite = useSelector( getSelectedSite );
+		const redirect = useSelector( ( state ) => getDomainRedirect( state, selectedSite?.domain ) );
+
+		return <Wrapped { ...props } key={ `redirect-${ redirect.targetHost }` } />;
+	},
+	'withRedirectAsKey'
+);
```

Comment:
Don't think we need this, it was on the original but didn't do anything. I suspect old code or maybe old usage required this.

Reply:
removed